Integrations/Panorama/Panorama.py

```diff
@@ -3410,48 +3410,67 @@ def panorama_get_traffic_logs_command():
 ''' Logs '''
 
 
+def build_array_query(query, arg_string, string, operator):
```

Comment:
how do you handle a single parameter?

Reply:
@yuvalbenshalom  argToList will still make a list of one item in it.
and it's handled in the function.